java/src/com/mongodb/csfle/CSFLEHelpers.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.csfle;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.bson.BsonBinary;
+import org.bson.BsonDocument;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.AutoEncryptionSettings;
+import com.mongodb.ClientEncryptionSettings;
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.IndexOptions;
+import com.mongodb.client.model.vault.DataKeyOptions;
+import com.mongodb.client.vault.ClientEncryption;
+import com.mongodb.client.vault.ClientEncryptions;
+
+/*
+ * Helper methods and sample data for this companion project.
+ */
+public class CSFLEHelpers {
```

Comment:
I have a similar concern as Matt Broadstone raised [here](https://github.com/mongodb-university/csfle-guides/pull/3#discussion_r350221795).  Do we want to bury this important code in a helper class, where it might just be indiscriminately copied rather than understood?

Reply:
The purpose of this is to produce a downloadable example that requires a minimal amount of user interaction to get CSFLE running so that they can observe it in action. The Use Case guide (https://docs.mongodb.com/ecosystem/use-cases/client-side-field-level-encryption-guide/) that this project accompanies steps the user through each section to build understanding. If we were to map these to the "5-E's of Education", this project provides "Engage" and a tad of "Explore", and the CSFLE guide provides "Explain".

Was there something in particular that you wanted to change the structure of or separate?

Reply:
Pulling it out into a separate helper class with static methods makes it more obvious that it's not a well-designed abstraction.  But I'm willing to let it go.

Reply:
Resolving this as this was our intention; can change it later if needed.